lib/screens/signup/signup_repository.dart

```diff
@@ -22,3 +28,42 @@ class FakeSignupRepository implements SignupRepository {
     }
   }
 }
+
+class APISignupRepository implements SignupRepository {
+  final String classTag = "APILoginRepository";
+  @override
+  Future<void> signup(
+      String firstName, String lastName, String email, String mobileNumber, String password) async {
+    final String tag = classTag + "login";
+    http.Response response;
+    try {
+      response = await sl.get<http.Client>().post(
+        S.registerUrl,
+        body: <String, dynamic>{
+          S.firstnameKey: firstName.toString(),
```

Comment:
`firstName` and all the others below are already `Strings`.

Reply:
made the change